app/src/main/java/com/example/trempelapp/dataLayer/network/NetworkSource.kt

```diff
@@ -0,0 +1,5 @@
+package com.example.trempelapp.dataLayer.network
+
+interface NetworkSource {
```

Comment:
I think it's not correct naming. You have to call it exactly what it should do. F.e. if this service is responsible for user endpoint call it either UserService or UserApi.

Reply:
And I cannot find Retrofit generator class. Did you miss it?

Reply:
done